addonscript/script.py

```diff
@@ -40,6 +40,9 @@ def get_default_config(base_dir=None):
 
     signed_addon_url = await retry_async(
         get_signed_addon_url, args=(context, locale, upload_data['pk']),
+        attempts=10,  # 14 attempts with default backoff yield around 10 minutes of time
```

Comment:
14 attempts? attempts is set to 10; bad comment, or bad setting?

Reply:
bad comment, fixed.